cypress/Dockerfile

```diff
@@ -2,9 +2,10 @@ FROM cypress/included:5.2.0
 
 WORKDIR /app
 
-RUN npm install --save-dev cypress-axe axe-core
+RUN npm install --save-dev cypress-axe@0.9.1 axe-core cypress-cucumber-preprocessor cosmiconfig
```

Comment:
how come this was pinned at 0.9.1?

Reply:
Because this version of cypress-axe plays nice with cypress-cucumber, and later versions fight with it. We hope the problem in later versions of cypress-axe will get fixed, but if not, there is a kludgey workaround we could do if we needed a later cypress-axe to work